src/data/content/design-system/componenti/datepicker.yaml

```diff
@@ -70,7 +70,11 @@ tabs:
             title: Quando usarlo
             headingLevel: 2
             text: |
-              _In stesura_
+              Utilizza il componente Datepicker quando gli utenti hanno bisogno di un calendario, ad esempio per programmare un evento o per sapere il giorno della settimana. Questo facilita la scelta della data.
+
+              ### Alternative
+
+              - Usa i componenti [input](/design-system/componenti/input) (mese e giorno in formato numerico) e [select](/design-system/componenti/select) (anno) nel caso di date familiari, come la data di nascita, e quando non è importante conoscere il giorno della settimana.
```

Comment:
Nel caso di date familiari agli utenti, come la data di nascita, e quando non è importante sapere il giorno della settimana, puoi usare il componente [input](/design-system/componenti/input) per il mese e il giorno in formato numerico e il componente [select](/design-system/componenti/select) per l'anno.